### PR TITLE
ci: check target branch only when PR is opened

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: .github/workflows/COMMENT_DEVELOP_FIX.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: COMMENT_DEVELOP_FIX
 on:
   pull_request:
+    types:
```
Comment (nikku) [inline .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: Do we want this? 

In this case we'd miss out on (future) cleaned-up commit history hints.

Comment (MaxTru) [reply .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: Did you test this before locally or so Maciej? I tried multiple cases locally and the behavior (at least for me) was not always predictable

Comment (barmac) [reply .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: I haven't, but we can use this branch for tests.

Comment (barmac) [reply .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: Let me have a test PR.

Comment (barmac) [reply .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: It works on opened: https://github.com/camunda/camunda-modeler/pull/2949

Comment (barmac) [reply .github/workflows/COMMENT_DEVELOP_FIX.yml:4]: But doesn't trigger a new comment on additional commits.

**Review by MaxTru — CHANGES_REQUESTED** [review]
~~See https://github.com/camunda/camunda-modeler/pull/2948#issuecomment-1121167898~~ Wait, this was wrong

**Review by MaxTru — APPROVED** [review]
I also tested with forcepush. Works.

Thanks @barmac :clap:

**Pull request merged**